### PR TITLE
Ss/dcos 49721

### PR DESCRIPTION
**Pull request by SarahStegall**
## Description
https://jira.mesosphere.com/browse/DCOS-49721
Further edits to DC/OS CLI pages for 1.12 based on CLI version 0.7.11.

## Urgency
- [x] Blocker 
- [ ] High
- [ ] Medium

### CODE REVIEW

File: pages/1.12/cli/command-reference/dcos-config/dcos-config-set/index.md [hunk 1]
```diff
@@ -9,37 +9,42 @@ enterprise: false
...
+
+| Name  | Value |
+|-----------------------|------------------------------------------------|
+| `core.dcos_acs_token`   | The DC/OS authentication token. When you log into the DC/OS CLI using `dcos auth login`, it stores the authentication token value locally. For more information, see [the IAM API](/1.12/security/oss/iam-api/). |
+| `core.dcos_url`         | The public master URL of your DC/OS cluster|
+| `core.mesos_master_url` | The Mesos master URL. Defaults to `core.dcos_url` |
+| `core.pagination`       | Indicates whether to paginate output. Defaults to true.|
+| `core.ssl_verify`       | Indicates whether to verify SSL certificates or set the path to the SSL certificates|
+| `core.timeout`          | The request timeout in seconds, with a minimum value of 1 second. Defaults to 3 minutes.|
+| `core.ssh_user` |   |
```
Comment (rgo3) [inline pages/1.12/cli/command-reference/dcos-config/dcos-config-set/index.md]:
```suggestion
| `core.ssh_user` | The user used when using `ssh` to connect to a node of your DC/OS cluster. Defaults to "core". |
```

File: pages/1.12/cli/command-reference/dcos-config/dcos-config-set/index.md [hunk 2]
```diff
@@ -9,37 +9,42 @@ enterprise: false
...
+| `core.ssh_proxy_ip`  |   |
```
Comment (rgo3) [inline pages/1.12/cli/command-reference/dcos-config/dcos-config-set/index.md]:
```suggestion
| `core.ssh_proxy_ip`  | The public IP of a node that will be used as a proxy when using ssh to connect to a private node of your DC/OS cluster |
```

File: pages/1.12/cli/command-reference/dcos-config/dcos-config-set/index.md [hunk 3]
```diff
@@ -9,37 +9,42 @@ enterprise: false
...
+|  `core.reporting` | |
```
Comment (rgo3) [inline pages/1.12/cli/command-reference/dcos-config/dcos-config-set/index.md]: I sadly have not figured out the last two here, maybe @ArmandGrillet can help us out or knows somebody?

**Comment by SarahStegall** [comment]
@ArmandGrillet Do you have any input on the dcos-config-set page? We are still missing definitions for `core.reporting` and `core.prompt.login`. Thanks.

File: pages/1.12/cli/command-reference/dcos-config/dcos-config-set/index.md (hunk 3 above)
Comment (ArmandGrillet) [reply pages/1.12/cli/command-reference/dcos-config/dcos-config-set/index.md]: `ssh_proxy_ip`: Whether to use a fixed ssh proxy host (Bastion) for node SSH access.
`reporting`: Whether to report usage events to Mesosphere.

**Comment by ArmandGrillet** [comment]
> @ArmandGrillet Do you have any input on the dcos-config-set page? We are still missing definitions for `core.reporting` and `core.prompt.login`. Thanks.

`core.reporting`: Whether to report usage events to Mesosphere.
`core.prompt.login`: Whether to prompt the user to log in when token expired; otherwise automatically initiate login.

**Pull request merged**